dockers/docker-base-arm64/Dockerfile.j2

```diff
@@ -0,0 +1,85 @@
+FROM multiarch/debian-debootstrap:arm64-jessie
```

Comment:
this is pretty much the same as existing docker-base, it is better to reuse the existing file and add $arch in the j2 template so that the dockerfile can be generated depending on the configured arch.

Reply:
The below the diff in docker-base:-

{code}
$ diff -Naur dockers/docker-base dockers/docker-base-arm64/
diff -Naur dockers/docker-base/Dockerfile.j2 dockers/docker-base-arm64/Dockerfile.j2
--- dockers/docker-base/Dockerfile.j2   2019-06-06 15:39:20.659871862 +0530
+++ dockers/docker-base-arm64/Dockerfile.j2     2019-06-07 21:59:05.814780451 +0530
@@ -1,4 +1,4 @@
-FROM debian:jessie
+FROM multiarch/debian-debootstrap:arm64-jessie

 ## Remove retired jessie-updates repo
 RUN sed -i '/deb http:\/\/deb.debian.org\/debian jessie-updates main/d' /etc/apt/sources.list
diff -Naur dockers/docker-base/sources.list dockers/docker-base-arm64/sources.list
--- dockers/docker-base/sources.list    2019-06-06 15:39:20.659871862 +0530
+++ dockers/docker-base-arm64/sources.list      2019-06-07 21:59:05.814780451 +0530
@@ -1,7 +1,5 @@
-## Debian mirror on Microsoft Azure
-## Ref: http://debian-archive.trafficmanager.net/
+deb http://archive.debian.org/debian jessie main contrib non-free
+deb-src http://archive.debian.org/debian jessie main contrib non-free
+deb-src http://archive.debian.org/debian jessie-backports main contrib non-free
+deb http://archive.debian.org/debian jessie-backports main contrib non-free

-deb http://debian-archive.trafficmanager.net/debian/ jessie main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian/ jessie main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian-security/ jessie/updates main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian-security/ jessie/updates main contrib non-free
{code}


#1 FROM tag can be used in j2 script

Reply:
or the debian repo changes, I think we could have options to choose, we do not need to use debian-archive.trafficmanager.net. we can use template to solve this problem instead of hardcode the mirror site into the dockerfile.

Reply:
It would have been easier for me to support ARCH changes, if the same template method was used between jessie and stretch

Reply:
Done, removed docker-base-<ARCH> and docker-base-stretch-<ARCH>